GettyImages.Api/SdkException.cs

```diff
@@ -26,11 +27,11 @@ internal SdkException(string message, Exception innerException, HttpStatusCode?
 
         public HttpStatusCode? StatusCode { get; set; }
 
-        internal static void GenerateSdkException(HttpResponseMessage httpResponse, string message = null)
+        internal static async Task GenerateSdkException(HttpResponseMessage httpResponse, string message = null)
```

Comment:
Seems like we should rename this to `GenerateSdkExceptionAsync`

Reply:
Done.